examples/stamped_bin/stamped_bin_test.go

```diff
@@ -22,7 +22,13 @@ import (
 )
 
 func TestStampedBin(t *testing.T) {
-	if stamp.BUILD_TIMESTAMP == stamp.NOT_A_TIMESTAMP {
+	if stamp.BUILD_TIMESTAMP == stamp.NOT_A_TIMESTAMP || stamp.BUILD_TIMESTAMP == "" {
```

Comment:
(that said, `BUILD_TIMESTAMP` should always exist, I'm guessing, so it might be nice to continue to test that both stamping codepaths are actually passing it through and not just passing an empty `x_def`)